python/tank/commands/cache_yaml.py

```diff
@@ -82,7 +82,7 @@ def _run(self, log):
             yaml_cache.g_yaml_cache.get(path)
 
         items = yaml_cache.g_yaml_cache.get_cached_items()
-        pickle_path = os.path.join(root_dir, "yaml_cache.pickle")
+        pickle_path = self.tk.pipeline_configuration.get_yaml_cache_location()
```

Comment:
Make the get_yaml_cache_location public on the pipeline config object. There's no harm doing it since PipelineConfiguration is part of our internal API anyway.